### ISSUE TITLE
play toolbar broken, tempo not clickable or selectable with keyboard

### ISSUE THREAD

**Issue by 2-www** [musescore/MuseScore#23739]
### Issue type

Accessibility issue (e.g. for keyboard-only or screen reader users)

### Description with steps to reproduce

1. create a new file in musescore
2. try to change tempo in the toolbar

time is drawn over buttons, clicking the tempo selects the time, tab skips over it

### Supporting files, videos and screenshots

![screenshot](https://github.com/user-attachments/assets/d5b5a2fb-5e4d-4243-8efe-4ccf42a47733)

### What is the latest version of MuseScore Studio where this issue is present?

4.3.2

### Regression

I was unable to check

### Operating system

Debian

### Additional context

default ui scaling (100%), reproducible with both ukrainian and english language set, and with all settings reset to defaults, and with layout reset to default

### Checklist

- [X] This report follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting bugs and issues
- [X] I have verified that this issue has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar issues
- [X] I have attached all requested files and information to this report
- [X] I have attempted to identify the root problem as concisely as possible, and have used minimal reproducible examples where possible

**Comment by cbjeukendrup** [comment]
This issue shows some visual similarity to https://github.com/musescore/MuseScore/issues/21036, but that one doesn't mention the fact that tab skips over it. 
The conclusion of that issue though is that it is specific to the FlatPak build, over which we don't have control. Reportedly, this bug does not occur in the "official" AppImage from musescore.org, and that might have to do with the fact that the AppImage used a different Qt 5 version than the FlatPak.

I'm interested how this will be after we release version 4.4, because that uses Qt 6.

**Comment by MarcSabatella** [comment]
I think there is some confusion here.  That display is read--only - it's not a way to set the tempo.  Setting the tempo is done by adding tempo marking to the score itself.

However, if you undock the play panel, it does expose a control that provides a temporary override to the actual tempo.  And that control is inaccessible as far as I know, because I don't think there is a keyboard-only way to undock the toolbar.

While we *could* add a keyboard command to undock a focused toolbar, I don;t think this is really the best solution.  Really. that temporary tempo override shouldn't require undocking - that's a super commjin complaint about MU4.  The problem arose only because there was an oversight in the original toolbar layout, where it wasn't realized the MU3 had an override.  So the undock hack was kind of tacked on to provide a quick workaround.  But it's high time we solve this for real, by making that override directly available within the toolbar.

**Comment by bkunda** [comment]
@2-www could you please add some further reproduction steps to this? It's not exactly clear from the report how you are trying to change the tempo.

Also a screen recording would be very helpful.

**Comment by cbjeukendrup** [comment]
Okay, so to summarise, there are two things here:
1. The time digits appear offset, but that part is a duplicate of https://github.com/musescore/MuseScore/issues/21036 and according to that issue it only occurs in third-party created builds and not in our AppImage, so we don't really have control over that
2. You would like to access the tempo slider via a popup triggered by the tempo display. I think that's a very reasonable request, and in fact I almost implemented that but never fully finished it. When I have a moment, I'll pick that up again and create a PR for it.
